### ISSUE TITLE
MacOS media keys not working

### ISSUE THREAD

**Issue by jcs-lambda** [clangen/musikcube#539]
Musikcube version: 0.98.0 #6a440c69
Installed via brew
OS: MacOS Monterey v 12.4 (M1)

Terminals:
 - built in Terminal.app: 2.12.7 (445)
 - alacritty: 0.10.1 (1)
 - wezterm: 20220624-141144-bd1b7c5d

Media keys do not work with musikcube. Any media key I press launches the Apple Music app.
Media keys do work with other apps when running (vlc, youtube in browser, tiny player)

Logs when run in Terminal & alacritty contain:
```
16:49:37 [info] [LOG SESSION] ---------- START ----------
16:49:37 [info] [PluginFactory] loading plugins
16:49:37 [info] [PluginFactory] loaded: /opt/homebrew/share/musikcube/plugins/libsupereqdsp.dylib
16:49:37 [info] [PluginFactory] loaded: /opt/homebrew/share/musikcube/plugins/libmacosmediakeys.dylib
...
16:49:37 [error] [macosmediakeys] failed to register media keys listener
16:49:37 [info] [macosmediakeys] unregistered media keys
...
```

Logs when run in Wezterm:
```
17:01:34 [info] [LOG SESSION] ---------- START ----------
17:01:34 [info] [PluginFactory] loading plugins
17:01:34 [info] [PluginFactory] loaded: /opt/homebrew/share/musikcube/plugins/libsupereqdsp.dylib
17:01:34 [info] [PluginFactory] loaded: /opt/homebrew/share/musikcube/plugins/libmacosmediakeys.dylib
...
17:01:35 [info] [macosmediakeys] listening to media keys
...
17:01:36 [info] [macosmediakeys] unregistered media keys
```

In Terminal and Alacritty, the plugin fails to load. In Wezterm, it loads, but still doesn't work.

The first time running musikcube in any of these terminal causes the system to pop up a "this app wants to control your computer" warning and I allow the requested "accessibility" permission (System Preferences -> Security & Privacy -> Privacy -> Accessibility). After that, still the media keys do not work. I have restarted the terminals, computer, etc.

Any ideas for other things I could try to get this to work?

Found[ this late addition to a closed issue](https://github.com/clangen/musikcube/issues/123#issuecomment-640125017) but didn't see any other mentions of media keys not working in MacOS.

**Comment by jondoe1337** [comment]
Not sure if this actually helps, but in iTerm2 the media keys work flawless.

**Comment by jcs-lambda** [comment]
Thanks for the heads up. I tried it out and experienced the same issue. I figured something must be wrong on my end, so I looked a little closer.

The actual media keys do work in Wezterm and ITerm2. Haven't tested Alacritty / Terminal.

I am using bluetooth headphones with touch controls for playback control. Headphone control (which I assumed / had read used the media key interface) does not work in either terminal. Seems to be an issue with the headphones using something besides the mediakey interface, although they work with other apps.
